apps/engmanagement/src/pages/api/skill/[...skillRecordings].ts

```diff
@@ -0,0 +1,124 @@
+import {NextApiRequest, NextApiResponse} from 'next'
+import {init} from '../../../skill/core/init'
+import renderPage from '../../../skill/core/pages'
+import {
+  OutgoingResponse,
+  SkillRecordingsAction,
+  SkillRecordingsHandlerParams,
+  SkillRecordingsOptions,
+  SkillRecordingsRequest,
+  SkillRecordingsResponse,
+} from '../../../skill/core/types'
+
+async function SkillRecordingsHandler<
```

Comment:
this belongs entirely to us and sends a page or a redirect back to our wrapper that builds the response below